src/Serenity.Scripts/CoreLib/UI/Helpers/UploadHelper.ts

```diff
@@ -23,8 +23,8 @@ export namespace UploadHelper {
             pasteZone: options.zone,
             done: function (e: JQueryEventObject, data: any) {
                 var response = data.result;
-                if (response.Error != null) {
-                    Q.alert(response.Error.Message);
+                if (!response.Error) {
+                    Q.notifyError(response.Error.Message);

```

Comment:
Somehow this `Q` gives error on typescript file. Do you get same error? Can you check it?
For fixing this you can add import on top of file `import { notifyError } from "../../Q";` then remove `Q.` in this line. After that, no body will faced with this error. 

![image](https://user-images.githubusercontent.com/2023440/115531183-28065180-a29d-11eb-80a2-d07f959a7f1b.png)